src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1064,6 +1063,7 @@ private void initConfigFromFlags(String[] args, PrintStream out, PrintStream err
       // For backwards compatibility, allow both commonJsPathPrefix and jsModuleRoot.
       List<String> moduleRoots = new ArrayList<>(flags.commonJsPathPrefix);
       moduleRoots.addAll(flags.moduleRoot);
+      moduleRoots.add(ES6ModuleLoader.DEFAULT_FILENAME_PREFIX);
```

Comment:
If the user is actually passing flags on the command line, I would expect they would include the default path (`./`) explicitly. So I'm not sure we actually want this line, except in the case when flags.moduleRoot is empty.